docs/examples/example_figure.py

```diff
@@ -0,0 +1,83 @@
+from screens.secspec import SecondarySpectrum
+from screens.visualization import make_sketch
+
+
+def gen_magnifs(th):
+
+    # Complex magnifications amplitudes of the scattering points
+    sig = 1.5*u.mas
+    a = 0.3*np.exp(-0.5*(th/sig)**2).to_value(1)
+    np.random.seed(2)
+    magnifs = a * np.random.normal(size=(th.size, 2)).view('c16').squeeze(-1)
+    magnifs[np.where(th == 0)] = 1
+    magnifs /= np.sqrt((np.abs(magnifs)**2).sum())
+
+    return magnifs
+
+
+def example_figs(th_par, magnifs,
+                 d_eff=2.*u.kpc, mu_eff=50.*u.mas/u.yr, fobs=330.*u.MHz,
+                 delta_f=1.*u.MHz, delta_t=20.*u.minute, nf=201, nt=101,
+                 **kwargs):
+
+    # Observation frequencies and times
+    f = fobs + np.linspace(-0.5*delta_f, 0.5*delta_f, nf)
+    t = np.linspace(-0.5*delta_t, 0.5*delta_t, nt)[:, np.newaxis]
+
+    # Create dynamic wavefield and dynamic spectrum
+    th_perp = np.zeros_like(th_par)
+    dynwave = dynamic_field(th_par, th_perp, magnifs, d_eff, mu_eff, f, t)
+    axes = tuple(range(0, dynwave.ndim-2))
+    dynwave = dynwave[...].sum(axes)
+    dynspec = np.abs(dynwave)**2
+
+    ss = SecondarySpectrum.from_dynamic_spectrum(dynspec, f=f, t=t, noise=0.,
+                                                 d_eff=d_eff, mu_eff=mu_eff,
+                                                 theta=th_par,
+                                                 magnification=magnifs)
+    tau = ss.tau
+    fd = ss.fd
+    secspec = np.abs(ss.secspec)**2
+
+    # Figure
+    plt.figure(figsize=(12., 10.))
+    plt.subplots_adjust(wspace=0., hspace=0.3)
+
+    # Plot dynamic spectrum
+    ax1 = plt.subplot(221)
+    ds_extent = (t[0][0].value, t[-1][0].value, f[0].value, f[-1].value)
+    im1 = ax1.imshow(dynspec.T, origin='lower', aspect='auto',
+                     extent=ds_extent, cmap='Greys')
+    plt.title('dynamic spectrum')
+    plt.xlabel('time $t$ [%s]' % t.unit.to_string('latex'))
+    plt.ylabel('frequency $f$ [%s]' % f.unit.to_string('latex'))
+    cbar = plt.colorbar(im1)
+    cbar.set_label('normalized intensity')
+
+    # Plot secondary spectrum
+    ax2 = plt.subplot(223)
+    ss_extent = (fd[0][0].value, fd[-1][0].value, tau[0].value, tau[-1].value)
+    im2 = ax2.imshow(secspec.T, origin='lower', aspect='auto',
+                     extent=ss_extent, cmap='Greys',
+                     norm=colors.LogNorm(vmin=1.e-4, vmax=1.))
+    plt.title('secondary spectrum')
+    plt.xlabel(r"differential Doppler shift $f_\mathrm{{D}}$"
+               rf"({fd.unit.to_string('latex')})")
+    plt.ylabel(r"relative geometric delay $\tau$ "
+               rf"({tau.unit.to_string('latex')})")
+    plt.xlim(-20., 20.)
+    plt.ylim(-35., 35.)
+    cbar = plt.colorbar(im2)
+    cbar.set_label('normalized power')
+
+    # Add sketch to the right
+    ax3 = plt.subplot(122)
+    make_sketch(th_par, rotation=-90.*u.deg, ax=ax3, **kwargs)
+    plt.title('schematic')
+
+    plt.show()
```

Comment:
My sense would be to also include code to generate an actual example figure, i.e.,
```
if __name__ == '__main__':
    <code for, say, the multi-point final figure>
```

Reply:
Done.

Reply:
Hmm, not actually done!

Reply:
@mhvk Sorry about that Marten, I'm still getting used to the github interface.

I'd made the changes locally and wrote these comments to tick them off here, but I should only do that after pushing the changes.

The most recent commit should have implemented all your comments.